client/state/data-layer/wpcom/sites/stats/google-my-business/test/index.js

```diff
@@ -0,0 +1,97 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { GOOGLE_MY_BUSINESS_STATS_SEARCH_REQUEST } from 'state/action-types';
+import { fetchGoogleMyBusinessStats, receiveStats } from '../';
+import { receiveGoogleMyBusinessStats } from 'state/google-my-business/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+describe( '#fetchGoogleMyBusinessStatsSearch', () => {
+	test( 'should dispatch HTTP request to Google My Business stats endpoint', () => {
+		const action = {
+			type: GOOGLE_MY_BUSINESS_STATS_SEARCH_REQUEST,
```

Comment:
Let's use the right action type here, i.e. `GOOGLE_MY_BUSINESS_STATS_REQUEST`.

Reply:
I'm surprised that these tests don't fail as this action type isn't used in reducers - this looks suspicious to me.